ocaml/testsuite/tests/parsetree/source_jane_street.ml

```diff
@@ -450,8 +450,8 @@ val f1 : ('a : value_or_null). local_ 'a -> unit @@ global many = <fun>
 val f2 : unit -> unit @@ global many = <fun>
 |}]
 
-module type S = sig
-  module type S0 = sig
+module type S = sig @@ portable contended
+  module type S0 = sig @@ portable contended
```

Comment:
Shouldn't this change the output below?

Reply:
No, this is just checking the roundtrip-ness of user-syntax print back, so it's working fine.